src/pdfboxing/util.clj

```diff
@@ -0,0 +1,38 @@
+(ns pdfboxing.util)
+
+(defn first-line-valid?
```

Comment:
That makes sense, and compressing this to a single regular expression would make it difficult to give informative error messages.